sdk/documentintelligence/Azure.AI.DocumentIntelligence/api/Azure.AI.DocumentIntelligence.netstandard2.0.cs

```diff
@@ -70,9 +70,9 @@ public partial class AnalyzedDocument
         public System.Collections.Generic.IReadOnlyDictionary<string, Azure.AI.DocumentIntelligence.DocumentField> Fields { get { throw null; } }
         public System.Collections.Generic.IReadOnlyList<Azure.AI.DocumentIntelligence.DocumentSpan> Spans { get { throw null; } }
     }
-    public partial class AnalyzeDocumentRequest
+    public partial class AnalyzeDocumentContent
```

Comment:
Had to rename those "Request" suffixes to make our analyzer stop complaining. It's a change we would do for the next version anyway, so not a big deal.